src/Libraries/DSCPython/CPythonEvaluator.cs

```diff
@@ -231,6 +235,32 @@ static CPythonEvaluator()
             }
         }
 
+        /// <summary>
+        /// Processes additional bindings that are not actual inputs.
+        /// Currently, only the node name is received in this way.
+        /// </summary>
+        /// <param name="scope">Python scope where execution will occur</param>
+        /// <param name="bindingNames">List of binding names received for evaluation</param>
+        /// <param name="bindingValues">List of binding values received for evaluation</param>
+        private static void ProcessAdditionalBindings(PyScope scope, IList bindingNames, IList bindingValues)
+        {
+            string nodeName;
+            if (!bindingNames[0].Equals("Name"))
```

Comment:
can we extract this to a const somewhere?

Reply:
I could define a const, but where do I put it? DSCPython and DSIronPython don't have anything in common now with PythonNodeModels...